CHANGELOG.asciidoc

```diff
@@ -38,6 +38,11 @@ Notes:
 [float]
 ===== Bug fixes
 
+* Fix instrumentation of `mongodb` to avoid multiple command handler
+  registrations when client is created via `MongoClient.connect` static
+  method. ({pull}3586[#3586])
+
+
```

Comment:
This should be added to a new `        ==== Unreleased` block.

Reply:
If https://github.com/elastic/apm-agent-nodejs/pull/3590 goes in first and this merges in those changes, then this might be solved.